react/modules/module04/README.md

```diff
@@ -1,47 +1,45 @@
-# Неделя 4
+# Week 4
 
 ## React. Hooks
 
 
 
 
-!! Задание выполняется в школьном приватном репозитории !!
-Нужно выполнять задание на базе предыдущего
+!! The task must be done in the school’s private repository !! 
+Do the task on the basis of the previous one
 
 
 
 
-Что должно быть сделано:
+What must be done:
 
 
 
-1) Создать отдельную ветку для этого задания
-2) Переписать основную страницу на хуки сохранив функционал. Все тесты по итогу должны работать
-3) Переписать форму на React Hook Form, сохранив функционал. Все тесты по итогу должны работать
+1) Create a separate branch for this task
+2) Saving functionality, rewrite the main page with the hooks. All the test should be working in the end
+3) Saving functionality, rewrite the form with React Hook Form. All the test should be working in the end
```

Comment:
it's okay, can we rewrite the last sentences as All tests should be

Reply:
Is "All the tests should be successfully completed" better?

Reply:
Just All tests